docs/docs/Development.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "Development"
+linkTitle: "Development"
+weight: 3
+description: >
+  How to develop Triggers and Functions
+---
+
+When the project is small it is acceptable to have a single file that contains the entire code base. But as the project grows and become complex it is less comfortable to maintain it as a single file project. Lucky for us JS already face such problem and has the relevant tool to maintain a multi file project and compress it on build time to a single file that contains all the code. In this tutorial we will explain how to create a multi file project and how to wrap it as a single file and send it to Triggers and Functions. The tutorial assume you have Redis with the RedisGears 2.0 module installed on `localhost:6379`. See [getting started](../README.md) section for installation instructions.
```

Comment:
We should probably change this entire taturial to use the sdk: https://github.com/RedisGears/RedisGearsJSSDK

Reply:
I updated it with gears API